app/uk/gov/hmrc/taxenrolmentassignmentfrontend/controllers/SignInAgainController.scala

```diff
@@ -16,33 +16,71 @@
 
 package uk.gov.hmrc.taxenrolmentassignmentfrontend.controllers
 
-import play.api.Logging
 import play.api.i18n.I18nSupport
 import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import play.api.{Logger}
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
 import uk.gov.hmrc.taxenrolmentassignmentfrontend.controllers.auth.AuthAction
+import uk.gov.hmrc.taxenrolmentassignmentfrontend.logging.EventLoggerService
+import uk.gov.hmrc.taxenrolmentassignmentfrontend.logging.LoggingEvent._
+import uk.gov.hmrc.taxenrolmentassignmentfrontend.repository.SessionKeys.REDIRECT_URL
+import uk.gov.hmrc.taxenrolmentassignmentfrontend.repository.TEASessionCache
+import uk.gov.hmrc.taxenrolmentassignmentfrontend.services.{EACDService, UsersGroupSearchService}
 import uk.gov.hmrc.taxenrolmentassignmentfrontend.views.html.SignInAgain
 
 import javax.inject.{Inject, Singleton}
-import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.ExecutionContext
 
 @Singleton
 class SignInAgainController @Inject()(
```

Comment:
I think that this signout function is still need as there are links of other pages to signout (i.e PTOnAnotherAccount).  I think what you have done needs to be in a new controller specific for SigninAgainWithSAAccount

Reply:
Sorry ignore that just realised it is the SignOutController that is called in other places so overwriting that controller is fine just maybe rename it :-)

Reply:
I made changes to the controller and view name to be along the lines of `SignInWithSAAccount`